internal/tableprinter/tableprinter_options.go

```diff
@@ -0,0 +1,49 @@
+package tableprinter
+
+import "fmt"
+
+// TablePrinterOption is a type of func(*TablePrinter).
+type TablePrinterOption func(*TablePrinter) error
+
+// WithOutputFormat returns a function func(opts *TablePrinter) that sets `format` in TablePrinter pointer instance.
+func WithOutputFormat(format TableOutputFormat) TablePrinterOption {
+	return func(opts *TablePrinter) error {
+		opts.format = format
+		return nil
+	}
+}
+
+// WithOutputFormatFromString returns a function func(opts *TablePrinter) that sets `format` in TablePrinter pointer instance of type `TableOutputFormat` from string.
```

Comment:
Great! All we need to do here is line-wrap all comments to 80 characters.

Reply:
Done 👍🏽